### PR TITLE
Change overview map to OSM from Mapbox due to expired token

### PR DESCRIPTION
**Pull request by digital-codes**

### CODE REVIEW

File: themes/codefor-theme/layouts/partials/map-overview.html [hunk 1]
```diff
@@ -52,14 +52,23 @@
       map.addControl(L.control.zoom({ position: 'topright' }));
 
       // basemap
+      // NOTE: Do not use Mapbox with time limited token!
+      var cfgUrl='https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png';
+	    var cfgAttrib='Map data © <a href="https://openstreetmap.org">OpenStreetMap</a> contributors';
+	    var mapboxLayer = new L.TileLayer(cfgUrl, {	minZoom: 5, attribution: cfgAttrib});	
+      /*
       var mapboxLayer = L.tileLayer(
```
Comment (k-nut) [inline themes/codefor-theme/layouts/partials/map-overview.html:60]: Can your remove this section that is commented out now completely please?

File: themes/codefor-theme/layouts/partials/map-overview.html [hunk 2]
```diff
@@ -52,14 +52,23 @@
       map.addControl(L.control.zoom({ position: 'topright' }));
 
       // basemap
+      // NOTE: Do not use Mapbox with time limited token!
```
Comment (k-nut) [inline themes/codefor-theme/layouts/partials/map-overview.html:55]: No need to add a comment here. The reason for switching can go into the commit message :)

**Review by k-nut — CHANGES_REQUESTED** [review]

File: themes/codefor-theme/layouts/partials/map-overview.html (hunk 2 above)
Comment (digital-codes) [reply themes/codefor-theme/layouts/partials/map-overview.html]: Comments removed

**Comment by k-nut** [comment]
Looks good to me. 👍

**Review by k-nut — APPROVED** [review]

**Pull request merged**